### ISSUE TITLE
Hover event on flag series with "useHTML: true"

### ISSUE THREAD

**Issue by xSirrioNx** [highcharts/highcharts#6303]
#### Expected behaviour
Flag should be higlighted (changed border) and tooltip should be displayed.

#### Actual behaviour
Hover event doesn't work at all. 

#### Live demo with steps to reproduce
[http://jsfiddle.net/yz61co92/1/](http://jsfiddle.net/yz61co92/1/)

In flag serises A and B "useHTML" flag set in "true". In flag serie C it set to "false"
Hover on A and B series doesn't work, but on C works well.

P.S.: In my project i have little difference of this issue. I use icons in flags and hover event works only if you  hover on the edges of flag (on svg element) and don't work if you hover on icon (HTML element).

**Comment by pawelfus** [comment]
Hi @xSirrioNx 

Thank you for reporting this. 

*Internal note:*
We had similar issue with `dataLabels` and useHTML before: #5947 - probably flags are a different a bit. @KacperMadej will you take a look?

**Comment by KacperMadej** [comment]
@TorsteinHonsi For this issue only one of two commits is added to master branch and it's causing error in tests for `unit-tests/series-flags/usehtml`.

**Comment by karolkolodziej** [comment]
@MFMarkus can you please elaborate or share a demo where it is visible?
I've tried using the original config but everything seems to be fine- [demo](https://jsfiddle.net/BlackLabel/n6w2xaqz/).

**Comment by MFMarkus** [comment]
@karolkolodziej 

Sorry, I was a bit quick on this. The issue only manifested when I enabled useHTML which made me jump on it quickly. However after some digging I have discovered that it is indeed working outside my project. I finally discovered an old css class lying around:

`.highcharts-point > .highcharts-label-box {
    display: none;
  }`
  
 This class, together with useHTML, apparantly caused the issue but without breaking the flag during non HTML usage.
 
 Thanks for quick response and sorry for not digging properly!
